name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/dashboard/PerformanceCalculationWidget.java

```diff
@@ -180,17 +184,72 @@ public void update(ClientPerformanceSnapshot snapshot)
             case REDUCED:
                 fillInReducedValues(snapshot);
                 break;
+            case NONNEUTRAL:
+                fillInNonNeutralValues(snapshot);
+                break;
             default:
                 throw new IllegalArgumentException();
         }
 
         container.layout();
     }
 
+    private void fillInNonNeutralValues(final ClientPerformanceSnapshot snapshot)
+    {
+        
+        List<ClientPerformanceSnapshot.Category> categories = snapshot.getCategories();
+        
+        // header
+        labels[0].setText(categories.get(0).getLabel());
```

Comment:
**Anfangswert** als Überschrift passt nicht mehr, oder? Es wird wir ja gar kein Anfangswert mehr angezeigt.

Entweder die Überschrift ganz weglassen (dann muss man an der Formatierung rumspielen), oder vielleicht "Performance-relevante Bewegungen" und in der Summenzeile nur noch den berechneten Wert?